src/main.js

```diff
@@ -104,4 +121,14 @@ function createNewCover() {
   titles.push(inputTitle.value);
   descriptors.push(inputDescriptor1.value);
   descriptors.push(inputDescriptor2.value);
+  //Save created Cover to hidden array
+  hiddenCover = createdCover;
+};
+
+//Checks user's cover for duplicates before adding to Saved Covers array
+var hiddenCover;
+function saveCover() {
+  if (!savedCovers.includes(hiddenCover)){
+    savedCovers.push(hiddenCover);
+  };
```

Comment:
Hey @rwilliams659, one of our biggest issues is figuring out a way to perform this task (making sure duplicate cover instances aren't saved to the Saved Covers array) without using a global variable (we have one on line 129). Let us know if you have any ideas!